### PR TITLE
Remove zip and libz from provides.conf

### PR DESCRIPTION
**Pull request by moths**
Remove from provide.conf and therby use the oe-lite build zip and libz versions.

zip and libz is not directly mentioned in the setup/* files so no cleanup needed there.

### CODE REVIEW

File: conf/provided/all.conf [hunk 1]
```diff
@@ -15,10 +15,11 @@ native:m4 \
 native:git native:cvs native:svn native:mercurial native:bzr \
 native:autoconf native:automake native:gnu-config \
 native:bc \
-native:gzip native:bzip2 native:tar native:unzip native:zip native:xz \
+native:gzip native:bzip2 native:tar native:unzip native:xz \
 native:patch native:quilt native:diffstat \
 native:perl native:perl-runtime \
 native:python-runtime \
+native:ruby \
```
Comment (esben) [inline conf/provided/all.conf]: This is not mentioned in commit message.  Is it added on purpose?

File: conf/provided/all.conf [hunk 2]
```diff
@@ -28,4 +29,5 @@ native:mtd-utils-mkfs-jffs2 \
 native:wget \
 native:gawk native:sed native:grep \
 native:libiconv \
+native:libacl1-dev \
```
Comment (esben) [inline conf/provided/all.conf]: This is not mentioned in commit message.  Is it added on purpose?

File: conf/provided/all.conf (hunk 1 above)
Comment (kimrhh) [inline conf/provided/all.conf]: "conf/provided/all: Add ruby, build time dependency for qtwebkit" :)

Comment (esben) [inline conf/provided/all.conf]: Hehe, didn't see that there were more than 1 commit.

File: conf/provided/all.conf (hunk 2 above)
Comment (moths) [inline conf/provided/all.conf]: ok removed, it is relate to native:fakeroot. Without it fakeroot ends up building a filesystem with wrong permissions. Causing fx. /usr to only work for root, not normal users. But it belongs in another patch
